better-politics-mod/common/on_actions/BPM_code_on_actions.txt

```diff
@@ -262,9 +262,13 @@ on_bpm_monthly_pulse_country = {
 			}
 		}
 		if = {
```

Comment:
We can probably put this logic in `on_character_ig_membership` in the trade unions IG file, rather than on monthly pulse. See dev chat for details